.gitignore

```diff
@@ -22,4 +22,5 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
-.eslintcache
```

Comment:
Could you add back this line to the .gitignore? The eslintcache is actually a temporary file generated with ESLint when linting the code. We don't need it in the repo since it changes constantly! Thanks for removing the package-lock.json from the .gitignore by the way 😄